<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ID_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Segoe UI&quot;;
font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ID_textInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisconnectButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="Details_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Details_textInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="State_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="State_textInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Timestamp_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Timetamp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="None_checkbox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="started_checkbox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Since custom RP started</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Localtime_checkbox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Your local time</string>
    </property>
   </widget>
   <widget class="QLabel" name="Largeimage_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Large Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="Smalimage_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Small Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="Keylarge_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="Keysmall_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="keylarge_input">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="keysmall_input">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Textlarge_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Textlarge_input">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Textsmall_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Textsmall_input">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Buttonone_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Button 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Buttontwo_label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Button 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Textone_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Textone_input">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>580</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Texttwo_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>580</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Texttwo_input">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>580</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Urlone_input">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>630</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Urltwo_input">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>630</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Urltwo_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>630</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QLabel" name="Urlone_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Url</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Updatepresence_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>700</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Update Presence</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1249</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
